src/IGC/IGCWriter.cpp

```
@@ -124,7 +131,8 @@ IGCWriter::StartDeclaration(const BrokenDateTime &date_time,
   if (date_time.IsPlausible()) {
     char buffer[64];
     FormatIGCTaskTimestamp(buffer, date_time, number_of_turnpoints);
-    WriteLine(buffer);
+    if (buffer[0] != '\0')
+      WriteLine(buffer);
```

Cppcheck (2.20.0)

[error] Uninitialized variable

(uninitvar)